diff -urN gnochm-0.9.4.orig/data/Makefile.am gnochm-0.9.4/data/Makefile.am
--- gnochm-0.9.4.orig/data/Makefile.am	2005-02-13 12:39:30.267337344 +0100
+++ gnochm-0.9.4/data/Makefile.am	2005-02-13 12:42:26.926481072 +0100
@@ -44,8 +44,8 @@
 install-data-local:
 	if test x$(USE_NLS) = "xyes"; then \
 		for m in $(man_locales) ; do \
-			$(mkinstalldirs) $(locale_mandir)/$$m/man1; \
-			$(INSTALL_DATA) gnochm.1.$$m $(locale_mandir)/$$m/man1/gnochm.1 ; \
+			$(mkinstalldirs) $(DESTDIR)$(locale_mandir)/$$m/man1; \
+			$(INSTALL_DATA) gnochm.1.$$m $(DESTDIR)$(locale_mandir)/$$m/man1/gnochm.1 ; \
 		done \
 	fi
 	if test -z "$(DESTDIR)" ; then \
